src/backends/backend/triton_backend_manager.h

```diff
@@ -143,9 +144,19 @@ class TritonBackendManager {
       const BackendCmdlineConfig& backend_cmdline_config,
       std::shared_ptr<TritonBackend>* backend);
 
+  static std::unique_ptr<
+      std::unordered_map<std::string, std::vector<std::string>>>
+  BackendState();
+
+  TritonBackendManager(){};
```

Comment:
Why is this needed as it is empty? It it is needed add it in the private section as:
TritonBackendManager() = default

Reply:
If I don't add this, the deleted copy constructor will be picked up as the default constructor instead of this constructor.